tests/fixtures.py

```diff
@@ -210,28 +210,43 @@ def inception_typesystem_xml(inception_typesystem_path):
 def tokens(small_typesystem_xml):
     typesystem = load_typesystem(small_typesystem_xml)
 
+    cas = Cas(typesystem)
```

Comment:
Why do you create a cas here and then dont use it?

Reply:
The tokens are added to it and as part of this, the sofa is set on the tokens.